src/annotation_tool/project.py

```diff
@@ -113,14 +114,15 @@ def save_annotations(self):
         """
         Exports the project's annotations to its tab separated value (tsv) file.
         """
-        with open(self.tsv_file, "w") as file:
-            file.write("client_id\tpath\tsentence\tup_votes\tdown_votes\tage\tgender\taccent\n")
+        with open(self.tsv_file, "w", newline='', ) as file:
+            writer = csv.writer(file, delimiter='\t')
+            writer.writerow(self.TSV_HEADER_MEMBERS)
             for annotation in self.annotations:
                 properties = []
                 for key in self.TSV_HEADER_MEMBERS:
                     properties.append(
                             str(getattr(annotation, key)).replace("\n", "\\r"))
```

Comment:
It might be worth implementing this as a function in annotation.  So this would change to `properties.get_dict()` together with `csv.DictWriter`